packages/config/config/devices/0x0460/qnsw-001P16.json

```
@@ -29,31 +28,58 @@
 			"maxNodes": 9
 		}
 	},
+
 	"paramInformation": [
 		{
 			"#": "1",
 			"label": "Sw1 Switch Type",
```

```suggestion
			"label": "SW1 Switch Type",
```
Parameter labels should be **Title Case**
https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide?id=labels